dockerfile/3.1.1/Dockerfile

```diff
@@ -23,5 +23,7 @@ RUN set -x \
 	&& mv * /var/www/html \ 
 	&& cd /var/www/html \
 	&& touch config.php \
-	&& chmod 0777 config.php
-	
\ No newline at end of file
+	&& chmod 0777 config.php \
+	&& cron \
+	&& echo "*/15 * * * * root /usr/bin/php /var/www/html/cron/status.cron.php" >> /etc/crontab
```

Comment:
You hard coded the path to the file, but this is different for a lot of people. Could you make this variable?

Reply:
As this is inside the Docker container, and as it is built upon `eboraas/apache`, it should be the same for everyone. The Dockerfile is built only before pushed to dockerhub.

Reply:
The time to run is hardcoded too...
It could use some like environment variable

Reply:
@luiscoms Any change you could add this?

Reply:
@TimZ99 Sure, I can make the time configurable, but do you really want the path to be configurable as well? Considering it is inside the Docker container and will always be the same for everyone.